.ci/optimade-version.json

```diff
@@ -0,0 +1,6 @@
+{
+  "schemaVersion": 1,
+  "label": "OPTiMaDe",
+  "message": "v0.10.1",
```

Comment:
Technically, this is still on v0.10.0, as v0.10.1 has not yet been officially released.